src/components/shared/Modal.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../../styles/scss/modal.scss';
```

Comment:
Next pull request this change your scss file like so:

modal.moudle.scss 

So it will read import modal from '../../styles/scss/modal.module.scss

Then any of your styles for className will need to be 
{modal.*styleNamehere*}

Reply:
Okay, will do it.